src/commonMain/resources/shader-libraries/built-in/libraries/color_functions.glsl

```diff
@@ -0,0 +1,23 @@
+// Some commonly used color functions
+// Ben Bartlett
+
+/* Convert HSV values ot RGB values
+   Hue: cycles over range betwee 0.0 and 1.0; modulo 1
+   Saturaton: 0-1
+   Value: 0-1
+   */
+vec3 hsv2rgb(in vec3 c){
```

Comment:
the comments are very useful. Are these functions available to call in any shader then?